packages/datadog-plugin-graphql/src/index.js

```diff
@@ -67,6 +69,7 @@ function createWrapParse (tracer, config) {
         setError(span, e)
         throw e
       } finally {
+        config.hooks.parse(span, document, operation)
```

Comment:
Is the operation needed here? In the plugin we work with one operation in particular for a given document, so we only extract that one, but all operations are already part of the document, so here it's redundant.

Reply:
Since the input is the source and the output is the document, maybe it would be more in line with `validate` if the arguments are `span, source, document`?

Reply:
Makes sense, will change.